kpops/components/base_components/kafka_connect.py

```diff
@@ -33,7 +33,7 @@
 
 
 class KafkaConnector(PipelineComponent, ABC):
-    type: ClassVar[str] = "kafka-connect"
+    type = "kafka-connect"
```

Comment:
The type `str` is not needed to be specified here.

Reply:
it's not, though? I don't understand if you are asking it as a question or making a statement 😄